nucypher/blockchain/eth/sol/source/contracts/Adjudicator.sol

```diff
@@ -187,24 +187,24 @@ contract Adjudicator is Upgradeable {
     }
 
     /// @dev the `onlyWhileUpgrading` modifier works through a call to the parent `verifyState`
-    function verifyState(address _testTarget) public {
+    function verifyState(address _testTarget) public override virtual {
```

Comment:
I understand why these functions are now `override`, but why `virtual`?

Reply:
Only for tests (see `AdjudicatorV2Mock` in `AdjudicatorTestSet`)

Reply:
Also it could be useful if we will inherit our contracts (but this is unlikely to be)